schema/types/CountrySubdivisionCode.schema.json

```diff
@@ -2,10 +2,9 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/types/CountrySubdivisionCode.schema.json",
   "title": "Type - Country Subdivision Code",
-  "description": "Type representation of the second part of an ISO 3166-2 country subdivision code",
+  "description": "State, province, or equivalent identifier required for an address in this country",
   "type": "string",
   "minLength": 1,
-  "maxLength": 3,
-  "pattern": "^[A-Z0-9]{1,3}$",
+  "pattern": "^[A-Z0-9]{1,}$",
```

Comment:
@arthur-clara @JSv4 should we eliminate the pattern entirely? Now that this isn't ISO, who is to say that the only characters are latin alphabet A-Z and digits 0-9